frontend/src/components/App/Home/config.ts

```diff
@@ -0,0 +1,6 @@
+/** Allow selecting multiple clusters on home page. */
+export const MULTI_HOME_ENABLED = import.meta.env.REACT_APP_MULTI_HOME_ENABLED === 'true' || true;
```

Comment:
`|| true` here makes MULTI_HOME_ENABLED value to be true in all cases

Reply:
I've know that `By default in this PR multi cluster is enabled.` is mentioned in the description, let's just not forget to turn it off before we merge

Reply:
Yeah, we can make the decision to release it (turn it on/off) closer to the release.